recipes/libtins/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conans import tools, CMake, ConanFile
+import os
+
+
+class LibTinsConan(ConanFile):
+    name = "libtins"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mfontanini/libtins"
+    description = "High-level, multiplatform C++ network packet sniffing and crafting library."
+    license = "BSD-2-Clause"
+    topics = ("pcap", "packets", "network", "packet-analyser", "packet-parsing", "libpcap", "sniffing")
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package", "cmake_find_package_multi"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_cxx11": [True, False],
+        "with_ack_tracker": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_cxx11": True,
```

Comment:
I need advise on default value for this option.

libtins readme says 
> libtins is noticeably faster if you enable C++11 support
So I want to set it to True by default, but it will require disabling it manually to support older C++.

Or is it preferable to have same defaults as source library, so disable it by default?

Thanks.

Reply:
I think let's have True by default.

You already have `with_ack_tracker` which is True by default, so `boost` is required and you will have to have at least C++11 (afaik).

And, generally speaking, C++11 is not too much to ask for :)

Reply:
This option should be removed